lib/api/2.0/nodes.js

```diff
@@ -61,24 +62,45 @@ var nodesGetPollersById = controller(function(req) {
 });
 
 var nodesGetWorkflowById = controller(function(req) {
-    return nodes.getNodeWorkflowById(req.swagger.params.identifier.value);
+    return workflowApiService.getWorkflowsByNodeId(req.swagger.params.identifier.value,
+                                                   req.query)
+        .then(function(workflows){
+            if(req.swagger.query && _(req.swagger.query).has('active')) {
+                return _(workflows)
+                    .filter(function(workflow) {
+                        return workflow.active() === req.swagger.query.active;
```

Comment:
@benbp I was trying to make this consistent with the way it is done in the workflows route.  I believe the original intent to do post filtering was so it would be more database agnostic